### DISCUSSION TITLE
The sensitivity of the sliding model

### DISCUSSION THREAD

**Discussion by Wheats** [google/filament#4716] [General]
The sensitivity of the sliding model

Comparing the effect of the ios sliding model, after adding the sky_box, the sliding sensitivity is too large and too fast

There is a way to reduce sensitivity？

**Comment by romainguy** [comment]
What "sliding model" are you talking about?

**Reply by Wheats** [reply]
Slide this model

**Reply by Wheats** [reply]
When I add a sky box and slide the model, the sky box rotates very fast

**Reply by Wheats** [reply]
So I think it’s a sensitivity issue
